app/src/main-process/menu/build-default-menu.ts

```diff
@@ -546,7 +547,25 @@ export function buildDefaultMenu({
     showLogsItem,
   ]
 
-  if (__DEV__) {
+  if (__RELEASE_CHANNEL__ === 'development' || __RELEASE_CHANNEL__ === 'test') {
+    if (__WIN32__) {
+      helpItems.push(separator, {
+        label: 'Command Line Tool',
+        submenu: [
+          {
+            label: 'Install',
+            click: emit('install-windows-cli'),
+          },
+          {
+            label: 'Uninstall',
+            click: emit('uninstall-windows-cli'),
+          },
+        ],
+      })
+    }
+  }
+
+  if (__RELEASE_CHANNEL__ === 'development' || enableTestMenuItems()) {
```

Comment:
Given the suggested change above could we use only the feature flag here and let its definition control the behavior?

```suggestion
  if (enableTestMenuItems()) {
```

Reply:
My initial thoughts was that since it was always enabled for development, then, not include it in the feature flag which is where we optionally change things. But, I do agree it is a bit cleaner this way. :)